NMRangeSlider/NMRangeSlider.m

```diff
@@ -501,6 +493,9 @@ -(void)layoutSubviews
     }
 
     self.trackBackground.frame = [self trackBackgroundRect];
+    self.trackBackground.image = self.trackBackgroundImage;
+    self.trackBackground.layer.cornerRadius = self.thickness/2;
```

Comment:
Cool to include rounded corners :-) I'm wondering if that should be its own property though. Then it can be also a simple rectangle of someone wants that. What do you think?